worlds/wargroove2/Rules.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+
+from BaseClasses import Region, Location
+from .Levels import Wargroove2Level
+from worlds.AutoWorld import LogicMixin
+
+
+class Wargroove2Logic(LogicMixin):
+    pass
+
+
+def set_rules(world, level_list: [Wargroove2Level],
```

Comment:
```suggestion
def set_rules(multiworld, level_list: [Wargroove2Level],
```
You are loading the multiworld in here, not the world class. Would heavily recommend either renaming this to multiworld here, or actually loading in your world class (and then getting rid of the other things here since you can get them all from the world class).